File: app/src/main/java/com/halfplatepoha/jisho/jdb/Entry.java
package com.halfplatepoha.jisho.jdb;

import io.realm.RealmList;
import io.realm.RealmObject;

/**
 * Created by surjo on 19/12/17.
 */

public class Entry extends RealmObject {

    public long entrySeq;

    public String japanese;

    public RealmList<String> pos;

    public RealmList<Gloss> glosses;

    public boolean common;

    public String furigana;

    public RealmList<String> tags;

    public RealmList<String> fields;

    public RealmList<String> kanjiTags;

    public RealmList<String> dialects;

    public RealmList<String> kanaTags;

    public RealmList<Entry> related;

    public String note;

}

File: app/src/main/java/com/halfplatepoha/jisho/v2/realm/Entry.kt
package com.halfplatepoha.jisho.v2.realm

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Entry: RealmObject() {

    @PrimaryKey
    var _id: Int = 0

    var entry: String? = null
    var furigana: String? = null
    var summary: String? = null
    var types: String? = null
    var frequency: Double? = null

}